### PR TITLE
Remove margin-bottom of combo child fields

### PR DESCRIPTION
**Pull request by truongwp**
Fixes https://github.com/Strategy11/formidable-pro/issues/3374

### CODE REVIEW

File: css/_single_theme.css.php [hunk 1]
```diff
@@ -73,6 +73,10 @@
 .<?php echo esc_html( $style_class ); ?> .form-field{
 	margin-bottom:<?php echo esc_html( $field_margin . $important ); ?>;
 }
+
+.<?php echo esc_html( $style_class ); ?> .frm_combo_inputs_container > .form-field{
```
Comment (Crabcyborg) [inline css/_single_theme.css.php:77]: @truongwp the CSS is built using two CSS files: custom_theme.css.php and this file. This file adds the rule for each style in the database, but that isn't necessary for this style at all (as it isn't a custom setting).

We should move it to custom_theme.css.php instead.

Comment (truongwp) [reply css/_single_theme.css.php:77]: I think that is the right place to add that CSS. The `margin-bottom` above it causes this issue. If that `if` block is not run, we don't need this CSS too.

I found that I'm missing `$important` too.

Comment (truongwp) [reply css/_single_theme.css.php]: I got it. It's updated now.

**Review by Crabcyborg — APPROVED** [review]
Thanks @truongwp this looks good!

**Pull request merged**